testplan/runnable/interactive/http.py

```diff
@@ -919,10 +919,10 @@ def _extract_entries(entry: Dict) -> Dict:
     """
     entries = {}
 
-    # NOTE: we assume "entries" is present, see application of the function
-    for child in entry["entries"]:
-        if child["category"] == ReportCategories.TESTCASE:
-            entries[child["name"]] = {}
+    # NOTE: we ignore assertions (DEFAULT category) and testcases
+    for child in entry.get("entries", []):
+        if child["category"] in ("DEFAULT"):
```

Comment:
I don't think we shall rely on the category of child. Can we check the category of the entry upfront, as the stop condition for this recursive function?

Reply:
Let me check I remember, having some weird scenario, but perhaps with the update to frontend that is gone as well.